### ISSUE TITLE
Running REXX containing dialogues should issue an error in case of user interaction

### ISSUE THREAD

**Issue by Rune-Christensen** [zowe/zowe-cli#1744]
We use the command "zowe zos-tso issue command "exec somerexx"" extensively, to reuse some of our existing REXX on Z.
This causes problems sometimes, when a legacy REXX calls another legacy REXX that calls another ..., and somewhere a panel pops up that require user interaction. 
This causes the the user session to hang, and can also cause problems with the z/OSMF task. 
The effect is that no more "zowe zos-tso issue command "exec" commands can be issued, until the user session is timed out.

We would like Zowe to somehow register that a user interaction was initiated, and terminate the call and return an error instead of just hanging.

Some of this could be solved by a timeout, that returns control to the script running the Zowe command, with a timeout message.
But this wont fix the session, that I would assume is still hung.

Is this a problem that should be addresed to IBM instead of Zowe?
It might be that this should be solved by the underlying REST interface?

**Comment by zFernand0** [comment]
This could be a z/OSMF enhancement request.
Ideally, z/OSMF would return information about whether or not the backend process is waiting for input.
CC: @Joe-Winchester

**Comment by zFernand0** [comment]
Hey @Joe-Winchester,
I hope that the SHARE event went smoothly for all Zowe-related presentations.
I remember you asked a few weeks back about possible z/OSMF enhancement requests.
I believe this will be a great one to have in a future version of z/OS.

We would love to have the ability to determine whether a TSO command is waiting for input, and we believe it might be possible for z/OSMF to know that before sending back the TSO response data.

**Comment by Joey-ibm** [comment]
Yes, this is a good z/OSMF enhancement request. @Rune-Christensen please open a RFE (Idea) requirement to z/OSMF REST TSO API. Thanks!
